lib/charms/mysql/v0/mysql.py

```diff
@@ -276,6 +282,8 @@ def configure_mysql_users(self):
         configure_users_commands = (
```

Comment:
@delgod , I had this talk already but I want to confirm with you if we are going to enable the `monitoring` user from the get go, as charm default, or if we should create it on demand when relating to the COS stack.

Reply:
I want to have special/system users always